app/modules/conversations/conversations_router.py

```
@@ -1,7 +1,7 @@
 from typing import List
 
 from fastapi import APIRouter, Depends, HTTPException, Query
-from fastapi.responses import StreamingResponse
+from fastapi.responses import StreamingResponse, JSONResponse
```

Ruff (0.8.2)

`fastapi.responses.JSONResponse` imported but unused

Remove unused import: `fastapi.responses.JSONResponse`

(F401)